app/code/Magento/InventoryApi/Test/_files/stocks_with_sales_channel.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+require 'stocks.php';
+
+use Magento\InventoryApi\Api\StockRepositoryInterface;
+use Magento\InventorySalesApi\Api\Data\SalesChannelInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use Magento\Framework\App\State;
+use Magento\Framework\App\Area;
+
+/** @var \Magento\Framework\App\ObjectManager */
+$objectManager = \Magento\Framework\App\ObjectManager::getInstance();
+
+/** @var  SalesChannelInterface */
+$salesChannel = Bootstrap::getObjectManager()->get(SalesChannelInterface::class);
+
+/** @var  StockRepositoryInterface */
+$stockRepository = Bootstrap::getObjectManager()->get(StockRepositoryInterface::class);
+
+/** @var State */
+$appState = Bootstrap::getObjectManager()->get(State::class);
+
+/** SalesChannelInterface */
+$salesChannel->setCode('base');
+$salesChannel->setType(SalesChannelInterface::TYPE_WEBSITE);
```

Comment:
The fact that this fixture relies on `SalesChannel` hints that it should be in the `InventorySales` module, same as implementation of the fix itself. Please verify if my observations are correct.